<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dashboard</class>
    <widget class="QMainWindow" name="Dashboard">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>794</width>
                <height>599</height>
            </rect>
        </property>
        <property name="palette">
            <palette>
                <active>
                    <colorrole role="Base">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>255</red>
                                <green>255</green>
                                <blue>255</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="Window">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>255</red>
                                <green>255</green>
                                <blue>255</blue>
                            </color>
                        </brush>
                    </colorrole>
                </active>
                <inactive>
                    <colorrole role="Base">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>255</red>
                                <green>255</green>
                                <blue>255</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="Window">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>255</red>
                                <green>255</green>
                                <blue>255</blue>
                            </color>
                        </brush>
                    </colorrole>
                </inactive>
                <disabled>
                    <colorrole role="Base">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>255</red>
                                <green>255</green>
                                <blue>255</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="Window">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>255</red>
                                <green>255</green>
                                <blue>255</blue>
                            </color>
                        </brush>
                    </colorrole>
                </disabled>
            </palette>
        </property>
        <property name="windowTitle">
            <string>MQTT Dashboard</string>
        </property>
        <widget class="QStackedWidget" name="centralWidget">
            <widget class="QWidget" name="disconnectedWidget">
                <layout class="QGridLayout" name="disconnectedGrid">
                    <item row="2" column="0">
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item row="2" column="1">
                        <widget class="QPushButton" name="statusIndicator">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>28</pointsize>
                                    <weight>75</weight>
                                    <bold>true</bold>
                                </font>
                            </property>
                            <property name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property name="text">
                                <string>Disconnected</string>
                            </property>
                            <property name="flat">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="2">
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item row="0" column="1">
                        <spacer name="verticalSpacer">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>20</width>
                                    <height>40</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item row="3" column="1">
                        <spacer name="verticalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Vertical</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>20</width>
                                    <height>40</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>794</width>
                    <height>21</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
                <addaction name="actionLoadConfig"/>
                <addaction name="actionSave"/>
                <addaction name="actionSaveAs"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="actionConnect"/>
            <addaction name="actionDisconnect"/>
            <addaction name="actionAddTopic"/>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
        <action name="actionLoadConfig">
            <property name="text">
                <string>Load Configuration</string>
            </property>
            <property name="enabled">
                <bool>false</bool>
            </property>
        </action>
        <action name="actionSave">
            <property name="enabled">
                <bool>false</bool>
            </property>
            <property name="text">
                <string>Save</string>
            </property>
        </action>
        <action name="actionSaveAs">
            <property name="enabled">
                <bool>false</bool>
            </property>
            <property name="text">
                <string>Save As</string>
            </property>
        </action>
        <action name="actionConnect">
            <property name="text">
                <string>Connect</string>
            </property>
        </action>
        <action name="actionDisconnect">
            <property name="text">
                <string>Disconnect</string>
            </property>
        </action>
        <action name="actionAddTopic">
            <property name="text">
                <string>Add Topic</string>
            </property>
        </action>
    </widget>
    <resources>
        <include location="../assets/icons.qrc"/>
    </resources>
    <connections/>
</ui>
